e2e_test/e2e_test.go

```diff
@@ -36,3 +38,54 @@ func TestSendCelo(t *testing.T) {
 	err = network.AwaitTransactions(ctx, tx)
 	require.NoError(t, err)
 }
+
+func TestSingleNodeManyTxs(t *testing.T) {
+	rounds := 100
+	txsPerRound := 5
+	accounts := test.Accounts(1)
+	gc := test.GenesisConfig(accounts)
+	gc.Istanbul.Epoch = uint64(rounds) * 50 // avoid the epoch for this test
+	network, err := test.NewNetwork(accounts, gc)
+	require.NoError(t, err)
+	defer network.Shutdown()
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+
+	for r := 0; r < rounds; r++ {
+		txs := make([]*types.Transaction, 0, txsPerRound)
+		for j := 0; j < txsPerRound; j++ {
+			tx, err := network[0].SendCelo(ctx, common.Address{}, 1)
+			require.NoError(t, err)
+			require.NotNil(t, tx)
+		}
+		err = network.AwaitTransactions(ctx, txs...)
+		require.NoError(t, err)
+	}
+}
+
+func TestManyNodes(t *testing.T) {
+	accounts := test.Accounts(4)
```

Comment:
4 seems a bit low to be classed as "many", 4 is the minimum number of nodes we could use to explore situations where we have byzantine or failed nodes and the network continues to work.

Reply:
So I tried setting the number higher and the test seems to get stuck, I fiddled about with the round change params and was able to get networks of size 9 at least passing sometimes but taking 40 seconds to get there. So I think just renaming this to something like `TestNetworkManyTxs` would be fine.